### ISSUE TITLE
Missing NPC and quest "Mist" in Teldrassil

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5142]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
There is no NPC "Mist" spawned at where it should be and thus the escort quest that it provides cannot be done.
**Expected behaviour**: Tell us what should happen instead
There should be a cat NPC "Mist" spawned at the northwestern area of The Oracle Glade near a group of harpies. The NPC should provide an escort quest that makes it follow you as you bring it back to NPC Sentinel Arynia Cloudsbreak.
**Steps to reproduce the problem**:

1. Go to the northwestern area of The Oracle Glade in Teldrassil.
2. Look around all the harpies clusters for the cat NPC "Mist"
3. The NPC is missing

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Quest:Mist
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @wrathofthebug,
He is there on PTR.
![image](https://user-images.githubusercontent.com/6797611/33482432-bb8306f4-d6a1-11e7-95f0-03b940b8fd15.png)

**Comment by shanorr** [comment]
He is not missing but he is stuck and will not move.
